pkg/controller/nodelifecycle/scheduler/taint_manager_test.go

```diff
@@ -111,6 +108,63 @@ func createNoExecuteTaint(index int) v1.Taint {
 	}
 }
 
+// Waits for delete actions to appear and then returns them
+func getDeleteActions(fakeClientset *fake.Clientset) []clienttesting.Action {
+	var actions []clienttesting.Action
+main:
+	for ; ; actions = fakeClientset.Actions() {
+		for _, action := range actions {
+			if action.GetVerb() == "delete" {
+				break main
+			}
+		}
+	}
+	return actions
+}
+
+// Waits for delete actions to appear and checks if pod names
+// are equal to those stated in actions. When there appeared
+// all the actions with stated pod names then those actions returned
+func getDeleteActionsWithCertainPods(fakeClientset *fake.Clientset, podNames []string) []clienttesting.Action {
+	var actions []clienttesting.Action
+
+	var podActionsReceived []string
+	isEqual := func(a []string, b []string) bool {
+	main:
+		for _, valueA := range a {
+			for _, valueB := range b {
+				if valueA == valueB {
+					continue main
+				}
+			}
+			return false
+		}
+		return true
+	}
+main:
+	for ; ; actions = fakeClientset.Actions() {
+	secondary:
```

Comment:
Again, don't use labeled loops unless they improve readability. It is not the case here IMHO.

Reply:
Fixed